conf/dev.yaml

```diff
@@ -1 +1,12 @@
 logType: '{{ENV.LOG_TYPE || "cli" }}'
+plugins:
+  hapi-log-response:
+    requests: '{{ envVars.logRequests }}'
+    includeEventTags: true
+    ignoreUnauthorizedTry: true
+    requestPayload: true
```

Comment:
any reason this is all in here? don't we just need requestPayload and requestHeaders?